### PR TITLE
SLVSCODE-17 Add support for Vue.js

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "sonarlint-vscode",
   "displayName": "SonarLint",
   "description":
-    "On-the-fly bug and quality issues detection for JavaScript, Python and PHP",
+    "On-the-fly bug and quality issues detection for JavaScript, TypeScript, Python and PHP",
```
Comment (vilchik-elena) [inline package.json]: @henryju should it be in different commit?

Comment (henryju) [reply package.json]: @vilchik-elena Indeed
